src/kalite-app.py

```diff
@@ -42,5 +42,7 @@ def request_portal(url):
 sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
 connection = sock.connect_ex(('127.0.0.1', port))
 
-request_portal('http://localhost:8008')
+# We use a custom URI that will be handled by chromium-browser-appmode
+# so that chromium-browser gets launched with --class and -app set.
+request_portal('webapp://org.learningequality.KALite@http://localhost:8008')
```

Comment:
I updated the PR with a slightly bigger change and tested it, and seems to work exactly as intended:
  * If no xdg-desktop-portal-gtk package is installed (no backend) the request in the portal frontend emits `2` as response code, and we fallback to using the browser.
  * If the xdg-desktop-portal-gtk package is installed but I change the URI to contain a scheme other than http, https or webapp, the app chooser dialog is shown. If the user cancels now, nothing else happens (no fallback to the browser).
  * If I leave the  xdg-desktop-portal-gtk package and keep our custom URI in place, then everything works as expected and KA Lite launches in chromium, in app mode.